resources/views/searches/tasks/buttons/edit_task.blade.php

```diff
@@ -0,0 +1,244 @@
+<!-- Edit user button - OPEN -->
+<span data-toggle="modal" href="#editTask-{{ $task->id }}">
+    <button type="button" class="btn btn-sm btn-outline-dark btn-margin"
+    data-toggle="tooltip" data-placement="top" title="{{ __('actions.edit') }}">
+        <span class="octicon octicon-pencil"></span>
+    </button>
+</span>
+<!-- Edit user button - CLOSE -->
+<!-- Edit user modal - OPEN -->
+<div id="editTask-{{ $task->id }}" class="modal fade">
+    <div class="modal-dialog modal-confirm">
+
+        <!-- Modal content - OPEN -->
+        <div class="modal-content">
+
+            <!-- Form edit user - OPEN -->
+            <form action="{{ Route('editTask', $task->id) }}" method="POST"
+            enctype="multipart/form-data">
+                @csrf
+                @method('PATCH')
+
+                <!-- Modal header - OPEN -->
+                <div class="modal-header">
+                    <h5 class="modal-title font-weight-bold ml-3">
+                        {{ __('forms.edit_task') }}:
+                    </h5>
+                    <button type="button" class="close" data-dismiss="modal" aria-hidden="true">
+                        &times;
+                    </button>
+                </div>
+                <!-- Modal header - CLOSE -->
+
+                <!-- Modal body - OPEN -->
+                <div class="modal-body text-center">
+                    <div class="row">
+                        <div class="col text-left">
+                            <label for="inputSector">
+                                {{ __('forms.sector') }}
+                            </label>
+                            <input type="text" name="Sector" class="form-control" value="{{ $task->Sector }}">
+                            @error('Sector')
+                                <p class="alert alert-danger">{{ $message }}</p>
+                            @enderror
+                        </div>
+
+                        <div class="col text-left">
+                            <label for="inputGroup">
+                                {{ __('forms.task_group') }}
+                            </label>
+                            <select type="text" name="Group" class="form-control">
+                                <option disabled selected>Choose an active group</option>
+                                <option>0</option>
+                                <option>1</option>
+                                <option>2</option>
+                                <option>3</option>
+                                <option>4</option>
+                                <option>5</option>
+                                <option>6</option>
+                                <option>7</option>
+                                <option>8</option>
+                                <option>9</option>
```

Comment:
Group is an integer input by the user, not a select, as it can be any positive number